servicetalk-http-api/src/test/java/io/servicetalk/http/api/AbstractHttpRequestMetaDataTest.java

```diff
@@ -79,13 +79,15 @@ public void testParseUriOriginForm() {
         assertEquals("foo=bar&abc=def&foo=baz", fixture.rawQuery());
         assertEquals("/some/path?foo=bar&abc=def&foo=baz", fixture.requestTarget());
 
-        assertNull(fixture.effectiveHost());
-        assertThat(fixture.effectivePort(), lessThan(0));
+        HostAndPort effectiveHostAndPort = fixture.effectiveHostAndPort();
```

Comment:
nit: This whole block could be improved as a single assertion that asserts before & after given host & port parameters.

Reply:
agreed, done.